### PR TITLE
elektra: update to 0.9.2

### PR DESCRIPTION
**Pull request by haraldg**
Notable changes:
* Move some plugins from libelektra-plugins to libelektra-cpp because
  they got reimplemented in C++.
* New package libelektra-zmq
* Move the crypto plugin from libopenssl to libgcrypt
* Disable host build: elektra doesn't need it anymore and nobody uses
  kdb during package builds for now.

Closes: #12307

Signed-off-by: Harald Geyer <harald@ccbib.org>

Maintainer: me
Compile tested: openwrt-sdk-mxs_gcc-8.4.0_musl_eabi.Linux-x86_64 from 2020-05-28
Run tested: none

### CODE REVIEW

File: libs/elektra/Makefile [hunk 1]
```diff
@@ -149,7 +146,7 @@ endef
 define Package/libelektra-crypto
   $(call Package/libelektra/Default)
   TITLE:=Elektra crypto plugin
-  DEPENDS:=+libelektra-core +libopenssl
+  DEPENDS:=+libelektra-core +libgcrypt
```
Comment (neheb) [inline libs/elektra/Makefile:149]: Curious why.

Comment (haraldg) [reply libs/elektra/Makefile:149]: It's an upstream decision. They don't support libopenssl anymore. Maybe @markus2330 can fill in the details?

It seems CI fails after 45 minutes, when it is still building boost. Just runs out of time?

I hope this is not a problem for merging the PR.

Comment (neheb) [reply libs/elektra/Makefile:149]: No it’s a RAM problem. I ran into the same issue locally with my VM. I’ll merge later today.

Comment (markus2330) [reply libs/elektra/Makefile:149]: We needed to put too much effort into maintenance, so we only kept the best plugin as evaluated by the author (@petermax2): https://www.libelektra.org/ftp/elektra/publications/nirschl2018cryptographic.pdf

**Pull request merged**